coriolis/conductor/rpc/server.py

```diff
@@ -2599,11 +2599,18 @@ def task_event(self, ctxt, task_id, level, message):
         db_api.add_task_event(ctxt, task_id, level, message)
 
     @task_synchronized
-    def task_progress_update(self, ctxt, task_id, current_step, total_steps,
-                             message):
+    def add_task_progress(self, ctxt, task_id, total_steps, message):
         LOG.info("Task progress update: %s", task_id)
-        db_api.add_task_progress_update(ctxt, task_id, current_step,
-                                        total_steps, message)
+        db_api.add_task_progress(ctxt, task_id, total_steps, message)
+
+    @task_synchronized
+    def update_task_progress(self, ctxt, task_id, step, total_steps, message):
+        LOG.info("Task progress update: %s", task_id)
```

Comment:
Please differentiate this log from the one in `add_task_progress_update` somehow.